core/field_angle.js

```diff
@@ -100,6 +100,31 @@ Blockly.FieldAngle.WRAP = 360;
  */
 Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
 
+
+/**
+ * Draws the border with the correct width.
```

Comment:
Comment updated.

Yes, in field.js the width is saved to the "size_.width" attribute.